alacritty.yml

```diff
@@ -22,6 +22,20 @@
   # check the local terminfo database and use `alacritty` if it is
   # available, otherwise `xterm-256color` is used.
   #TERM: alacritty
+  #
+  # WINIT_X11_SCALE_FACTOR
+  #
+  # This value is used to control scale factor used by terminal on X11, by
+  # default it automatically detects scale factor of each monitor and apply
+  # it dynamically by using XRandr. Supported values `randr` or floating point
+  # values, like `"1.0"`.
+  #WINIT_X11_SCALE_FACTOR: randr
```

Comment:
Why did you add this? I really don't want people looking at the config file and thinking it would be a good idea to change this to tweak their scale factor.

Reply:
I wasn't sure how to put it and how it should be discovered or not. Since we have certain values and I wasn't sure how to translate them.

Reply:
I think I'll remove env section bits.